packages/plugin-credentials/src/credentials/common.js

```diff
@@ -27,7 +27,7 @@ export default {
   session: {
     clientToken: makeStateDataType(Token, `token`).prop,
     isAuthenticating: {
-      default: false,
+      default: true,
```

Comment:
If I'm reading this right, isAuthenticating will *always* be true in node environments.